### PR TITLE
feat: Add params for github repo for Policy Sync PT

### PR DESCRIPTION
**Pull request by bryankaraffa**
### Description

Added parameters for the Policy Template Github Repo to enable this to be used for other repos without modifying Policy Template

### Link to Example Applied Policy

https://app.flexera.com/orgs/6/automation/applied-policies/projects/7954?policyId=6601da3c3ad5094c46961f0d

image

Example overriding defaults to use data from a public repo (this is for a customer PoC):

https://github.com/bryankaraffa/friendly-engine

### Contribution Check List

- [X] New functionality includes testing.
- ~[ ] New functionality has been documented in the README if applicable~
- [X] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: tools/policy_sync/policy_sync.pt [hunk 1]
```diff
@@ -1,67 +1,102 @@
 name "Policy Template Synchronization"
 rs_pt_ver 20180301
 type "policy"
-short_description "A policy to manage policy template. See the [README](https://github.com/flexera-public/policy_templates/tree/master/tools/policy_sync/) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
+short_description "This Policy Template can be used to synchronize (upload, overwrite, or alert) RS built-in policy templates in your account. See the [README](https://github.com/flexera-public/policy_templates/tree/master/tools/policy_sync/) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
```
Comment (XOmniverse) [inline tools/policy_sync/policy_sync.pt:4]: References to RightScale, such as RS, should be removed.

**Review by XOmniverse — CHANGES_REQUESTED** [review]
Only one minor change requested.

**Comment by bryankaraffa** [comment]
@XOmniverse -- if you have time for a re-review please and thank you !

**Review by XOmniverse — APPROVED** [review]
LGTM

**Pull request merged**